crates/bevy_audio/src/volume_level.rs

```diff
@@ -0,0 +1,154 @@
+/// Represents a volume level value.
+#[derive(Clone, Copy, Debug)]
+pub enum VolumeLevel {
+    /// Volume Amplitude Ratio (unitless)
+    Amplitude(f32),
+    /// Decibels of Amplitude Ratio (dB)
+    Decibels(f32),
```

Comment:
I can construct this with any valid float value, incl. NaN and Infinity. Maybe

- add the expected range to the doc comment
- or not make the variants public and add constructors
- or this normalized this on use and then warrants another doc comment :)

Reply:
Makes sense! From other Rust projects I'm used to quite high type safety so I was surprised to see this be unconstrained. Maybe an interesting idea to go thru the codebase and see where we can introduce some cheap constructors that make misuse less likely. I saw you already added a debug assert on output as well :)